python/semantic_kernel/connectors/search/bing/bing_search.py

```diff
@@ -1,214 +0,0 @@
-# Copyright (c) Microsoft. All rights reserved.
```

Comment:
What's the context for the search and search engine related deletions? I don't see any work item linked in the PR description.